test/e2e/ovn-ic/e2e_test.go

```diff
@@ -347,16 +347,27 @@ var _ = framework.SerialDescribe("[group:ovn-ic]", func() {
 func checkECMPCount(expectCount int) {
 	ecmpCount := 0
 	maxRetryTimes := 30
-	for i := 0; i < maxRetryTimes; i++ {
-		time.Sleep(3 * time.Second)
-		execCmd := "kubectl ko nbctl lr-route-list ovn-cluster "
-		output, err := exec.Command("bash", "-c", execCmd).CombinedOutput()
-		framework.ExpectNoError(err)
-		ecmpCount = strings.Count(string(output), "ecmp")
-		if ecmpCount == expectCount {
-			break
+	clusterNames := []string{"kind-kube-ovn", "kind-kube-ovn1"}
+
+	for _, clusterName := range clusterNames {
+		switchCmd := "kubectl config use-context " + clusterName
+		_, err := exec.Command("bash", "-c", switchCmd).CombinedOutput()
+		framework.ExpectNoError(err, "switch to kube-ovn cluster failed")
+
+		for i := 0; i < maxRetryTimes; i++ {
```

Comment:
```suggestion
		ginkgo.By("Checking logical router route count")
		for i := 0; i < maxRetryTimes; i++ {
```

Reply:
After I add these logs, e2e will hang, but it can be run without adding them. Is it because the resources are not enough?

https://github.com/kubeovn/kube-ovn/actions/runs/8323908381/job/22815427164

Reply:
There is another timeout failure: https://github.com/kubeovn/kube-ovn/actions/runs/8323633672/job/22774090635